binary_tree_service.py

```diff
@@ -0,0 +1,64 @@
+'''
+Created on 22 May 2017
+
+@author: Eva
+'''
+from binary_tree import BinaryTree
+
+def print_ancestors(tree, key):
```

Comment:
Your tests right now cannot really test that the right set of nodes is printed.

In order to deal with that, you can do the following:

* define a `get_ancestors` function that returns either the list of ancestors or `None` if the key is not in the tree.
* implement `print_ancestors` by calling `get_ancestors`.

You still are not actually testing `print_ancestors`, but you can fully test `get_ancestors`.